src/core/thread/mesh_forwarder.cpp

```diff
@@ -1254,6 +1259,13 @@ ThreadError MeshForwarder::SendFragment(Message &aMessage, Mac::Frame &aFrame)
 
         if (payloadLength > fragmentLength)
         {
+            if (!aMessage.IsLinkSecurityEnabled())
```

Comment:
Thanks for the hints @jwhui 
@jwhui @abtink , please help to review the updated code.